SubProject.cmake

```diff
@@ -225,8 +229,13 @@ if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/.gitsubprojects")
         "else()\n"
         "  file(APPEND .gitsubprojects \"git_subproject(${__subproject})\n\")\n"
         "endif()\n")
+     set(SUBPROJECT_PATHS "${SUBPROJECT_PATHS} ${__subproject_dir}" 
+       CACHE INTERNAL SUBPROJECT_PATHS)
```

Comment:
A slight better way is to set a global property, at least it's not kept between cmake runs:
```
    set_property(GLOBAL APPEND PROPERTY
                 ${PROJECT_NAME}_ALL_DEP_TARGETS ${LibName})
```